### ISSUE TITLE
Custom font size is limited between 13 and 20

### ISSUE THREAD

**Issue by Mikaela** [element-hq/element-web#23068]
### Steps to reproduce

1. Settings, UI/Appearance, Font size
2. The slider can go between 13 and 18
3. I want a smaller font so I click custom font size
4. I cannot go below 12 as it says that it must be between 12 and 18.

#### What did you expect?

I expected to  be able to specify as custom font size instead of being restricted to the slider above (which I thought allowed the same numbers, but apparently caps to 18 instead of 20).

#### What happened instead?

I am forced to stay between 13 and 20.

### Operating system

"Fedora release 36 (Thirty Six)"

### Browser information

Mozilla/5.0 (X11; Linux x86_64; rv:105.0) Gecko/20100101 Firefox/105.0 ID:20220812093714

### URL for webapp

develop.element.io

### Application version

Element-versio: 39eee10c576f-react-8db7766a4012-js-3f6f5b69c7a1 Olm-versio: 3.2.12

### Homeserver

pikaviestin.fi is on Synapse 1.64.0

### Will you send logs?

No

**Comment by Mikaela** [comment]
My real problem is that left sidebar is using too big font for me to be able to utilise it.

The workaround to this is setting *font size to 15* and set *page zoom level to 80 %* in the browser.

**Comment by andybalaam** [comment]
A possible workaround is to use my (unsupported, experimental) tampermonkey script here: https://gitlab.com/andybalaam/element-web-tweaks/

**Comment by Diablo-D3** [comment]
Any update on this? Seems like a simple fix to remove the check entirely.

**Comment by andybalaam** [comment]
@vector-im/product can someone comment on this? We currently prevent the user from choosing the font size they want, with a message "Custom font size can only be between 13 pt and 20 pt".

I think a warning might be more appropriate - this is actively preventing some people making the app more accessible for their needs.

**Comment by Diablo-D3** [comment]
This bug intersects also with how Element considers text size; this sets the base value, everything else seems to be relative offsets of this.

#23913 basically illustrates the problem where font size in the prefs does not directly set the message body text size. So, a lot of stuff is kinda wrong here, and limiting the user from fixing it is kinda pear-shaped.

**Comment by t3chguy** [comment]
Fixed by https://github.com/matrix-org/matrix-react-sdk/pull/12246
